GoogleSignIn/Sources/GIDKeychainHandler/API/GIDKeychainHandler.h

```diff
@@ -22,10 +22,13 @@ NS_ASSUME_NONNULL_BEGIN
 
 @protocol GIDKeychainHandler <NSObject>
 
+/// Loads the AuthState object from the keychain.
```

Comment:
Capitalizing "AuthState" makes it seem like a type. Either use `OIDAuthState` or "auth state."

Reply:
Done.